From d9b524704d504b8e29d0b6fff5f9daf2a042580c Mon Sep 17 00:00:00 2001
From: Quinn Jensen <quinn.jensen@freescale.com>
Date: Fri, 30 Jan 2009 15:33:16 -0700
Subject: [PATCH] ENGR00106063-1 ESDHC: Allow 2nd port to be disabled

On MX25, the ESDHC2 pins conflict with FEC, so allow it
to be disabled.

Signed-off-by: Quinn Jensen <quinn.jensen@freescale.com>
---
 drivers/mmc/host/Kconfig |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/drivers/mmc/host/Kconfig b/drivers/mmc/host/Kconfig
index 3acaf69..e763b52 100644
--- a/drivers/mmc/host/Kconfig
+++ b/drivers/mmc/host/Kconfig
@@ -150,6 +150,13 @@ config MMC_IMX_ESDHCI
 
 	  If unsure, say N.
 
+config MMC_IMX_ESDHCI_SELECT2
+	bool "Enable second ESDHCI port"
+	depends on MMC_IMX_ESDHCI && ARCH_MX25
+	default n
+	help
+	  Enable the second ESDHC port
+
 config MMC_IMX_ESDHCI_PIO_MODE
 	bool "Freescale i.MX Secure Digital Host Controller Interface PIO mode"
 	depends on MMC_IMX_ESDHC != n
-- 
1.5.4.4

